cudamapper/samples/sample_cudamapper.cpp

```diff
@@ -73,7 +73,8 @@ void process_batch(std::vector<IndexDescriptor>& query_index_descriptors,
                    const std::shared_ptr<const io::FastaParser> query_parser,
                    const std::shared_ptr<const io::FastaParser> target_parser,
                    DefaultDeviceAllocator allocator,
-                   bool print)
+                   bool print,
+                   bool sam_format)
```

Comment:
could we update this bool to be an enum for which type of format we want to print `PAF`, `SAM`, `BAM`?

Reply:
👍 done